.github/workflows/docs-issue.yml

```
@@ -5,36 +5,34 @@
 # To reduce barriers for keeping docs up to date
 
 # **when?**
-# When a PR is labeled `user docs` and is merged.  Runs on pull_request_target to run off the workflow already merged,
-# not the workflow that existed on the PR branch.  This allows old PRs to get comments.
+# When an issue is labeled `user docs` and is closed as completed.  Can be labeled before or after the issue is closed.
 
 
 name: Open issues in docs.getdbt.com repo when a PR is labeled
 run-name: "Open an issue in docs.getdbt.com for PR #${{ github.event.pull_request.number }}"
 
 on:
-  pull_request_target:
+  issues:
     types: [labeled, closed]
 
 defaults:
   run:
     shell: bash
 
 permissions:
-    issues: write # opens new issues
-    pull-requests: write # comments on PRs
-
+    issues: write # comments on issues
 
 jobs:
   open_issues:
-    # we only want to run this when the PR has been merged or the label in the labeled event is `user docs`.  Otherwise it runs the
+    # we only want to run this when the is closed as completed and the label `user docs` has been assigned.
```

Nit: missing word `issue`
```suggestion
    # we only want to run this when the issue is closed as completed and the label `user docs` has been assigned.
```